namespaces/live-1.cloud-platform.service.justice.gov.uk/pathfinder-dev/02-limitrange.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: LimitRange
+metadata:
+  name: limitrange
+  namespace: pathfinder-dev
+spec:
+  limits:
+  - default:
+      cpu: 360m
```

Comment:
I'd recommend increasing both cpu and memory limits here, to allow your containers to consume more resources if they need them. In particular, we've seen cases where containers are getting throttled on startup, resulting in the first readiness probes failing, and the container being restarted and ending up in a crash loop.
I'd recommend 1000m for CPU. 1024Mi seems reasonable for memory, but it depends on your workload.